sessionCheck.js

```diff
@@ -6,77 +6,160 @@
      * same user authenticated within this app (the OIDC Relying Party).
      * @module SessionCheck
      * @param {Object} config - configation needed for working with the OP
-     * @param {string} config.subject - The user currently logged into the RP
+     * @param {string} [config.subject] - The user currently logged into the RP. Either the subject or the idToken are required
+     * @param {string} [config.idToken] - The first id_token obtained as part of an interactive grant. Either the subject or the idToken are required
+     * @param {string} [config.responseType=id_token] - Response type to use to check the session. Supported options are id_token or none.
      * @param {string} config.clientId - The id of this RP client within the OP
      * @param {string} config.opUrl - Full URL to the OP Authorization Endpoint
      * @param {function} config.invalidSessionHandler - function to be called once any problem with the session is detected
-     * @param {string} config.redirectUri [sessionCheck.html] - The redirect uri registered in the OP for session-checking purposes
-     * @param {number} config.cooldownPeriod [5] - Minimum time (in seconds) between requests to the opUrl
-     * @param {string} config.scope [openid] - Session check scope; can be space-separated list
+     * @param {function} [config.initialSessionSuccessHandler] - optional function to be called after the first successful session check request.
+     * @param {function} [config.sessionClaimsHandler] - optional function to be called after every session check request. Only invoked when using responseType=id_token. Includes the claims and the count of session check requests that have been made so far.
+     * @param {string} [config.redirectUri=sessionCheck.html] - The redirect uri registered in the OP for session-checking purposes
+     * @param {number} [config.cooldownPeriod=5] - Minimum time (in seconds) between requests to the opUrl
+     * @param {string} [config.scope=openid] - Session check scope; can be space-separated list
```

Comment:
Jsdoc doesn't describe `ssoToken`, `ssoTokenName`, and `amUrl` params. 

On a separate but related note, it looks like this AM-specific technique is used under a single condition—when `config.ssoToken` is present; hence, it might make sense to introduce just one new param, an object; for example:
* @param {object} [config.amSession] - AM SSO configuration.
* @param {string} [config.amSession.ssoToken]
* @param {string} [config.amSession.ssoTokenName]
* @param {string} [config.amSession.url]